.env

```diff
@@ -53,6 +53,11 @@ EDM_AUTHENTICATIONS_PASSWORD__DEFAULT=4dm1n
 
 ACM_AUTHENTICATIONS_URI__DEFAULT=http://acm:5000/
 
+# OAuth client id and secret
+OAUTH_USER_EMAIL=oauth-user@wildme.org
+HOUSTON_CLIENT_ID=5ecba12d-d102-498e-805f-5531b448a67e
+HOUSTON_CLIENT_SECRET=6wql6OZ5TyRRDFFfC0SPhzFCyBXVQHVWOxgLBge24crbh4vuTQhOlTvedZ3Nay21
```

Comment:
If this is checked into git, and visible on gitlab, surely it's not secret?

Reply:
That's the names given to the strings for creating oauth clients -- client id and secret.

Anyway, this file is for development, we also put in things like `GITLAB_ADMIN_PASSWORD` and database passwords.  They should all be changed to something else for deployment.